react/features/recording/actions.web.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import React from 'react';
+
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { showNotification } from '../notifications';
+
+import { RecordingLimitNotificationDescription } from './components';
+
+export * from './actions.any';
+
+/**
+ * Signals that a started recording notification should be shown on the
+ * screen for a given period.
+ *
+ * @param {string} streamType - The type of the stream ({@code file} or
+ * {@code stream}).
+ * @param {string} participantName - The participant name that started the recording.
```

Comment:
There's no `participantName` parameter, as far as I see (am I missing something?).

Reply:
it was copy/paste error

Reply:
Fixed!